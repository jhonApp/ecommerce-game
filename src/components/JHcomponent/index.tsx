import * as S from './styles'

const JHcomponent = () => (
  <S.Wrapper>
    <h1>JHcomponent</h1>
  </S.Wrapper>
)

export default JHcomponent