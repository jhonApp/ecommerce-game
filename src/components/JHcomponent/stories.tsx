import { Meta, StoryObj } from '@storybook/react'
import JHcomponent from '.'

export default {
  title: 'JHcomponent',
  component: JHcomponent
} as Meta

export const Default: StoryObj = {}