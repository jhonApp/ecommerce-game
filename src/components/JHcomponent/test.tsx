import { render, screen } from '@testing-library/react'

import JHcomponent from '.'

describe('<JHcomponent />', () => {
  it('should render the heading', () => {
    const { container } = render(<JHcomponent />)

    expect(screen.getByRole('heading', { name: /JHcomponent/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})