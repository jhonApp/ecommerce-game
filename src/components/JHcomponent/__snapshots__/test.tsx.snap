// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<JHcomponent /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    JHcomponent
  </h1>
</main>
`;
